tests/auth_tests/test_views.py

```diff
@@ -752,6 +752,61 @@ def test_redirect_to_login_with_lazy_and_unicode(self):
         self.assertEqual(expected, login_redirect_response.url)
 
 
+class LoginRedirectAuthenticatedUser(AuthViewsTestCase):
+
+    def setUp(self):
+        super(LoginRedirectAuthenticatedUser, self).setUp()
+        self.dont_redirect_url = '/login/redirect_authenticated_user_default/'
+        self.do_redirect_url = '/login/redirect_authenticated_user/'
+
+    def test_default(self):
+        """ Keep on the login page by default """
```

Comment:
"Stay on ..."
(and please be consistent with no spacing around the sentences)
Add a period to each sentence too.